src/Functions.Worker.Extensions.OpenAI/ChatBot/ChatBotCreateOutputAttribute.cs

```diff
@@ -3,7 +3,7 @@
 
 using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
 
-namespace Functions.Worker.Extensions.OpenAI;
+namespace Functions.Worker.Extensions.OpenAI.ChatBot;
```

Comment:
We need to update the namespaces of all the classes we're adding to start with `Microsoft.Azure.`